### DISCUSSION TITLE
Unable to execute test case in coolwallet SDK

### DISCUSSION THREAD

**Discussion by samleong-wits** [CoolBitX-Technology/coolwallet-js-sdk#418] [Q&A]
1. Unable to run test case in Coolwallet SDK/ Unable to test after function added into SDK
Steps:
a. Run script-tester with port 8080
b. Npm test coin-terra returned with 404
![image](https://user-images.githubusercontent.com/104709835/168476592-fdca88de-677e-4ede-9cf1-7ad71965cbd9.png)

Question: What are steps in order to run npm test in coolWallet SDK?

2. How do we start the endpoint "http://localhost:8080", and what service is this referring to? 
![image](https://user-images.githubusercontent.com/104709835/168476478-4c027050-6c02-4376-96b4-369bee0df54e.png)

**Comment by hank121314** [comment]
The test cases in CoolWallet SDK are only for internal team members. You should use script-tester or coin-tester to test the changes.
